db/fixtures/miq_event_definitions.csv

```diff
@@ -70,6 +70,17 @@ request_host_shutdown,Host Shutdown Request,Default,host_operations
 request_host_reboot,Host Reboot Request,Default,host_operations
 request_host_enable_vmotion,Host Vmotion Enable Request,Default,host_operations
 request_host_disable_vmotion,Host Vmotion Disable Request,Default,host_operations
+
+#
+# Middleware Operation
+#
+hawkular_deployment,EAP Application Deployment,Default,middleware_operations
+hawkular_deployment_remove,EAP Application Undeployment,Default,middleware_operations
```

Comment:
Is "Undeployment" a word? @jdoyleoss @jstickler

Reply:
You could probably change it to "Undeploy EAP Application".

Reply:
I'm sure we want to start the event name with "EAP" so it's easily recognizable in the event lists (e.i. when they are ordered alphabetically)